src/mono/msbuild/AndroidAppBuilder/Templates/runtime-android.c

```diff
@@ -184,6 +185,12 @@ strncpy_str (JNIEnv *env, char *buff, jstring str, int nbuff)
         (*env)->ReleaseStringUTFChars (env, str, copy_buff);
 }
 
+int
+tkill(pid_t tid, int sig)
```

Comment:
shouldn't this be in the runtime somewhere?

Reply:
hm.. it shouldn't be needed at all for Android - it was removed from NDK

Reply:
then I'm even more confused what this does :D

Reply:
removed it so now we rely on `pthread_kill` on Android 32bit. Tests seem pass